packages/toolpad-app/next.config.mjs

```diff
@@ -198,6 +198,24 @@ export default /** @type {import('next').NextConfig} */ withSentryConfig(
           source: '/:path*',
           headers: securityHeaders,
         },
+        {
+          source: '/reactDevtools/:path*',
```

Comment:
This surprises me, aren't there only scripts hosted under this path?

Reply:
You are right, it works without, removed, let's keep it simple